### ISSUE TITLE
Documentation on installing jupyter widget

### ISSUE THREAD

**Issue by alxiang** [pytorch/captum#640]
Running the below steps to install the jupyter widget fail:

```
jupyter nbextension install --py --symlink --sys-prefix captum.insights.widget
jupyter nbextension enable captum.insights.widget --py --sys-prefix
```

This is the error message:
`ImportError: No module named widget`

The widget folder seems to be in captum/insights/attr_vis in this repo, and installing with the jupyter extension "captum.insights.attr_vis.widget" worked for me.

**Comment by satishjasthi** [comment]
Hi @bilalsal 

Is this issue still open

**Comment by NarineK** [comment]
It looks like this was addressed in #641. Closing this issue
